pkg/plugin/grpc.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
```

Comment:
I think the interface should be defined in `github.com/abcxyz/jvs/apis/v0` and let's combine `grpc.go` and `interface.go`

Reply:
Most of the stuff under github.com/abcxyz/jvs/apis/v0 are generated code with simple helper method, so I feel a bit weird to have this interface to sit together with other generated code. 

If I understand go-plugin usage correctly, interface.go is to define interface regardless of implementation protocol (i.e. it could be grpc, net/rpc, etc). I think it is cleaner to split them to have grpc in its own file.

Reply:
`apis/v0` is where we define all the external facing contracts. The plugin interface is one of those contracts. And because grpc will be the only plugin protocol, it's fine we also define it there. Having them in a single file under the same package makes it easy to reference.

Reply:
Split out https://github.com/abcxyz/jvs/pull/290 and added the interface there